src/ferc_xbrl_extractor/xbrl.py

```diff
@@ -231,11 +226,9 @@ def get_fact_tables(
     Data-Package descriptor describing the output database if requested.
 
     Args:
-        taxonomy_path: URL of taxonomy.
+        taxonomy_path: Zipfile with archived taxonomies for form.
```

Comment:
nit: This arg is called `taxonomy_source` now.

Reply:
Fixed!